### DISCUSSION TITLE
How do you redirect to your landing, blog and marketing pages ?

### DISCUSSION THREAD

**Discussion by aswinmohanme** [plausible/analytics#1593] [Anything Else]
I am building a platform as a Phoenix LiveView app. Currently, the landing page is served from Phoenix itself. I would like to host a blog, separate out the landing and marketing pages as a different static site generated by Next.JS or Jekyll. 

I saw that Plausible redirects to a landing page if not signed in, I expected `PageController` to have a redirect clause but found it to only render an `index.html`. How do you manage to redirect to the blog and the marketing pages from your Phoenix App, any pointers would be appreciated? Is it done on the ngnix level ?

Also great job with Plausible, I use it for the above-mentioned platform.

**Comment by ukutaht** [comment]
Hey @aswinmohanme 

>  Is it done on the ngnix level ?

Yes. We use Netlify but the idea is the same. The reverse proxy first checks if there's a static page that matches the url. If the page exists (e.g. `/` or `/blog`), it will serve a static page from [plausible/website](https://github.com/plausible/website). If the url does not match any static pages, the request is forwarded to the Phoenix app.

Hope that helps

**Reply by ukutaht** [reply]
In Nginx I've used `try_files` directive in the past to make that happen

**Reply by aswinmohanme** [reply]
Thanks! I found the _redirects file and I'm modelling something similar, although on Vercel.

**Reply by ukutaht** [reply]
Yup, that should work
